docs/TabBarExample.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) "../src/less/index.less";
+
+.TabBarExample {
+  .margin--top--2xl;
+  font-size: 0.875rem;
```

Comment:
We don't have a text--size for this, but I see it used semi-often. I wonder if we should make one or just enforce size--medium or size--small and remove this in between.